crnk-core/src/main/java/io/crnk/core/engine/internal/dispatcher/controller/ResourceIncludeField.java

```diff
@@ -23,13 +23,12 @@ protected void assertRequestDocument(Document requestDocument, HttpMethod method
 		}
 	}
 
-	protected RegistryEntry getRegistryEntry(String resourceType) {
+	protected RegistryEntry getRegistryEntry(String resource) {
```

Comment:
that should not be necessary, either it is resourcePath or resourceType to avoid mixing both up.

Reply:
Didn't get it, I understand that I have to check for the resourcePath and the resourceType, so having something generic like resource I think it's ok. Can you explain me a bit more what you are trying to say?

Reply:
I think you should no need a method that checks both and rather have two separate methods. Either something is resourcePath and getEntryByPath must be called. Or is a resourceType and getEntry is called.